android/src/main/java/com/pinmi/react/printer/RNNetPrinterModule.java

```diff
@@ -58,8 +58,13 @@ public void connectPrinter(String host, Integer port, Callback successCallback,
     }
 
     @ReactMethod
-    public void connectAndSend(String host, Integer port, String data, Callback successCallback, Callback errorCallback) {
+    public void connectAndSend(String host, Integer port, String data, String brand, Callback successCallback, Callback errorCallback) {
+        // this.init(successCallback, errorCallback);
```

Comment:
why is this commented?

Reply:
This should be removed. You can find similar line below it. which does initialization